### PR TITLE
Explain how async atoms trigger Suspense fallback

### PR DESCRIPTION
**Pull request by Noitidart**
close https://github.com/pmndrs/jotai/issues/276

### CODE REVIEW

File: docs/jotai/basics/async.mdx [hunk 1]
```diff
@@ -73,3 +73,41 @@ const Component = () => {
 ```
 
 > There's no way to know as of now if an atom suspends because of `read` or `write`.
+
+## Triggering `Suspense` fallback
```
Comment (dai-shi) [inline docs/jotai/basics/async.mdx:77]:
```suggestion
### Triggering `Suspense` fallback
```

This section applies only for "Async write atom" not "Async read atom" which works totally different with Suspense.

**Review by dai-shi — CHANGES_REQUESTED** [review]
Thanks for your work! I really hope someone to work on docs. Appreciated.

Does the comment make sense?

File: docs/jotai/basics/async.mdx (hunk 1 above)
Comment (Noitidart) [reply docs/jotai/basics/async.mdx:77]: :) 

Oh yes totally I'll update this now.

Comment (Noitidart) [reply docs/jotai/basics/async.mdx]: Changes comitted.

**Review by dai-shi — APPROVED** [review]

**Comment by Noitidart** [comment]
Thank you!

**Pull request merged**